Index: base-files-6.0.1-1canaima12/share/dot.bashrc
===================================================================
--- base-files-6.0.1-1canaima12.orig/share/dot.bashrc	2012-07-05 11:54:52.687860224 -0430
+++ base-files-6.0.1-1canaima12/share/dot.bashrc	2012-07-05 12:08:53.735359351 -0430
@@ -5,14 +5,27 @@
 # PS1='${debian_chroot:+($debian_chroot)}\h:\w\$ '
 # umask 022
 
-# You may uncomment the following lines if you want `ls' to be colorized:
-# export LS_OPTIONS='--color=auto'
-# eval "`dircolors`"
-# alias ls='ls $LS_OPTIONS'
-# alias ll='ls $LS_OPTIONS -l'
-# alias l='ls $LS_OPTIONS -lA'
-#
+# enable color support of ls and also add handy aliases
+if [ -x /usr/bin/dircolors ]; then
+    export LS_OPTIONS='--color=auto'
+    eval "`dircolors -b`"
+    alias ls='ls $LS_OPTIONS'
+    alias ll='ls $LS_OPTIONS -lh'
+    alias l='ls $LS_OPTIONS -CF'
+    alias grep='grep $LS_OPTIONS'
+fi
+
 # Some more alias to avoid making mistakes:
 # alias rm='rm -i'
 # alias cp='cp -i'
 # alias mv='mv -i'
+
+alias useradd='useradd -G lp,cdrom,floppy,audio,dip,dialout,video,plugdev,netdev,bluetooth,lpadmin,powerdev,scanner,users,fuse'
+
+# enable programmable completion features (you don't need to enable
+# this, if it's already enabled in /etc/bash.bashrc and /etc/profile
+# sources /etc/bash.bashrc).
+if [ -f /etc/bash_completion ]; then
+    . /etc/bash_completion
+fi
+
