Index: base-files-6.0.1-1canaima12/share/nsswitch.conf
===================================================================
--- base-files-6.0.1-1canaima12.orig/share/nsswitch.conf	2012-07-05 12:09:52.447361217 -0430
+++ base-files-6.0.1-1canaima12/share/nsswitch.conf	2012-07-05 12:10:11.691360396 -0430
@@ -7,6 +7,7 @@
 passwd:         compat
 group:          compat
 shadow:         compat
+sudoers:        compat
 
 hosts:          files dns
 networks:       files
